acceptance/pre-suite/01_install_rototiller.rb

```diff
@@ -8,8 +8,9 @@
   end
   scp_to(sut, gem_name, gem_name)
 
+  rake_version = `rake --version`.split[2]
   if ENV['RAKE_VER']
```

Comment:
It looks like you're only relying on the 'rake --version' command now... should we remove this if statement?